packages/component/src/ErrorBox.js

```diff
@@ -29,13 +29,10 @@ const ErrorBox = ({ error, type }) => {
 };
 
 ErrorBox.defaultProps = {
-  children: undefined,
```

Comment:
why are we removing these fields?

Reply:
This is a linting fix. `children` is never used in the component, and `error` is required so default value is not required. :)